serveur_calcul_python/tests/data_gen/regulation_data_store.py

```
@@ -0,0 +1,220 @@
+            1, # reg 3 1 signifie additoin 
+            None,# reg 4 se 1
+            3, # reg 4 ou plus contraignant
+            None, # reg 5 plafond max
+            3,
+            3,
+            None, # reg plafnd seuil
+            4,
+            4
+            ],
+        'cases_fix_min':[
+            0, # 
```

Ruff (0.15.12)

[warning] Line with empty comment

Delete the empty comment

(PLR2044)